src/cmem/ActivityControl/ActivityControlWidget.tsx

```diff
@@ -63,6 +61,10 @@ export interface IActivityControlProps extends TestableComponent {
      * only use necessary width, not always the available 100% of parent element
      */
     canShrink?: boolean;
+    /**
+     * if this is set the spinner is replaced when the progress has finished from 0 - 1
+     */
+    progressFinishedIcon?: JSX.Element;
```

Comment:
Please set the type so that only an icon element can be used here. You can do this by using something like `React.ReactElement<IconProps>` as type spec here.